src/wcm/__main__.py

```diff
@@ -166,4 +166,17 @@ def publish(component, profile="default", debug=False, dry_run=False):
 def download(id, profile="default",file_path = None):
     logging.info("Downloading component")
     _download.download(id, profile=profile,download_path=file_path)
-    print("done")
\ No newline at end of file
+    print("done")
+
+
+@cli.command(help="Lists all the components in the current wings instance")
+@click.option(
+    "--profile",
+    "-p",
+    envvar="WCM_PROFILE",
+    type=str,
+    default="default",
+    metavar="<profile-name>",
+)
+def list(profile="default"):
+    _list.list_components()
```

Comment:
`profile` should be passed down while calling `list_components`

Reply:
I'll fix this tomorrow, ive got a couple of other bug fixes/ modifications to do. Thanks for feedback